graphql/schema/types/project_settings.graphql

```diff
@@ -188,4 +188,5 @@ type ProjectAlias {
   taskTags: [String!]!
   variant: String!
   variantTags: [String!]!
+  parameters: [Parameter!]
```

Comment:
tbh I'm not sure what our stance is on nullable vs non-nullable fields but I like making lists non-nullable, unless there's a reason it should be null. i like it because on the frontend you won't have to check if the array is null before checking if it's empty ([ref](https://yelp.github.io/graphql-guidelines/nullability.html#lists))


```suggestion
  parameters: [Parameter!]!
```

Reply:
Good idea!